### DISCUSSION TITLE
Features and Usage

### DISCUSSION THREAD

**Discussion by mohanr** [haskell/haskell-language-server#4352] [General]
I enabled `company` and 'lsp` in Doom Emacs and LSP is ready as per the message but at this point I don't know how the HLS can help. I place a cursor on an item and I don't see the type( C-c C-d ). My OCaml LSP (assume it is the LSP that does it ) shows it though. I assume now I have to understand how the HLS can help. Is there documentation dealing with this ? 
The switches I usually use are different now ? 
Unsure if this is a Doom Emacs integration problem as I haven't use the HLS properly.
Code completion shows options and when I choose I see an error.

```
Error processing message (error "No plugins are available to handle this SMethod_CompletionItemResolve request.
 Plugins installed for this method, but not available to handle this request are:
ghcide-completions does not handle resolve requests for ghcide-completions: error decoding payload:expected Bool, but encountered Null).").
```

**Comment by michaelpj** [comment]
Have you looked through the troubleshooting documentation? https://haskell-language-server.readthedocs.io/en/latest/troubleshooting.html
